vsts/generate_javadocs_branch.ps1

```diff
@@ -96,6 +96,19 @@ function CreateJavadocReleaseBranch($GitHubName, $GitHubEmail, $Sources) {
     Copy-Item -Force -Path .\provisioning\security\x509-provider\* -Destination ..\provisioning\security\x509-provider
     Copy-Item -Recurse -Force -Path .\provisioning\security\x509-provider\com -Destination ..\provisioning\security\x509-provider
 
+    # Create the folder to place content in if it does not exist.
```

Comment:
After the maven command generates the content, move it to the corresponding folder. The name of the folder will be configured on the pipeline that runs this script.